File: TestGameForAndroidJava-master/app/src/main/java/com/example/testgamesource/MainActivity.java
package com.example.testgamesource;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.Window;
import android.view.WindowManager;

public class MainActivity extends AppCompatActivity {
    private Game game;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Window window = getWindow();
        window.setFlags(
                WindowManager.LayoutParams.FLAG_FULLSCREEN,
                WindowManager.LayoutParams.FLAG_FULLSCREEN
        );
        game = new Game(this);
        setContentView(game);
    }

    @Override
    protected void onPause() {
        game.pause();
        super.onPause();

    }
}

File: 3.5/Fragments/app/src/main/java/ru/samsung/itacademy/mdev/fragments/MainActivity.kt
package ru.samsung.itacademy.mdev.fragments

import android.content.Context
import android.content.res.Configuration
import android.graphics.Color
import android.os.Bundle
import android.widget.Button
import android.widget.LinearLayout
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import ru.samsung.itacademy.mdev.fragments.R.id
import ru.samsung.itacademy.mdev.fragments.R.layout


class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(layout.activity_main)
        var btn = findViewById<Button>(id.caption)
        var color1= findViewById<LinearLayout>(R.id.red);
        var color2= findViewById<LinearLayout>(R.id.blue);
        color1.setBackgroundColor(Color.RED)
        color2.setBackgroundColor(Color.BLUE);
        var flag = true;

        btn.setOnClickListener() {
            if(flag) {
                color1.setBackgroundColor(Color.BLUE);
                color2.setBackgroundColor(Color.RED);
                flag = false;
            }else{
                color1.setBackgroundColor(Color.RED)
                color2.setBackgroundColor(Color.BLUE);
                flag = true;
            }
            }
        }
    }


